### PR TITLE
Fix handlers to handle non-glob paths

### PR DESCRIPTION
**Pull request by darkbasic**

### CODE REVIEW

File: tests/loaders/schema/schema-from-typedefs.spec.ts [hunk 1]
```diff
@@ -51,6 +51,20 @@ describe('schema from typedefs', () => {
     expect(schema.getTypeMap()['Query']).toBeDefined();
   });
 
+  it('should work without globs correctly', () => {
```
Comment (kamilkisiela) [inline tests/loaders/schema/schema-from-typedefs.spec.ts]: Hello @darkbasic ! Could you also add the same test but in `integration.spec.ts`? So we know it won't break the chain of loaders and load schema correctly?

File: src/loaders/schema/schema-from-export.ts [hunk 2]
```diff
@@ -8,7 +8,13 @@ export class SchemaFromExport implements SchemaLoader {
   canHandle(pointerToSchema: string): boolean {
     const fullPath = isAbsolute(pointerToSchema) ? pointerToSchema : resolvePath(process.cwd(), pointerToSchema);
 
-    return isValidPath(pointerToSchema) && existsSync(fullPath) && extname(pointerToSchema) !== '.json';
+    if (isValidPath(pointerToSchema) && existsSync(fullPath) && extname(pointerToSchema) !== '.json') {
+      const exports = require(fullPath);
```
Comment (kamilkisiela) [inline src/loaders/schema/schema-from-export.ts]: I think it might break stuff when it's a `ts` file and no `ts-node/register` is used

Comment (kamilkisiela) [reply src/loaders/schema/schema-from-export.ts]: An example:

```ts
export const typeDefs: DocumentNode = gql`
  type Query { ... }
`;
```

**Comment by darkbasic** [comment]
Please remember to also merge https://github.com/ardatan/graphql-toolkit/pull/43

File: tests/loaders/schema/schema-from-typedefs.spec.ts (hunk 1 above)
Comment (darkbasic) [reply tests/loaders/schema/schema-from-typedefs.spec.ts]: Done

File: src/loaders/schema/schema-from-export.ts (hunk 2 above)
Comment (darkbasic) [reply src/loaders/schema/schema-from-export.ts]: Can you please elaborate? I used the very same logic used in the handler.

**Pull request merged**